core/src/main/java/com/kinandcarta/create/proxytoggle/core/intent/IntentUtils.kt

```diff
@@ -0,0 +1,19 @@
+package com.kinandcarta.create.proxytoggle.core.intent
+
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+
+fun getLaunchIntent(context: Context): Intent? {
+    return context.packageManager.getLaunchIntentForPackage(context.packageName)
+        ?.apply { flags = Intent.FLAG_ACTIVITY_NEW_TASK }
+}
+
+fun getPendingIntentFlags(): Int {
```

Comment:
Are these not pending intent flags you want for a specific use-case? Would you always want only these flags?

Might be better to keep this private to only where this is used?

Reply:
Only used in WidgetProvider currently, probably can make it private.

Reply:
done.